tests/certification/state/ravendb/ravendb_test.go

```diff
@@ -0,0 +1,255 @@
+package ravendb_test
+
+import (
+	"fmt"
+	"github.com/dapr/components-contrib/state"
+	stateRavenDB "github.com/dapr/components-contrib/state/ravendb"
+	"github.com/dapr/components-contrib/tests/certification/embedded"
+	"github.com/dapr/components-contrib/tests/certification/flow"
+	"github.com/dapr/components-contrib/tests/certification/flow/dockercompose"
+	"github.com/dapr/components-contrib/tests/certification/flow/network"
+	"github.com/dapr/components-contrib/tests/certification/flow/sidecar"
+	stateLoader "github.com/dapr/dapr/pkg/components/state"
+	daprTesting "github.com/dapr/dapr/pkg/testing"
+	daprClient "github.com/dapr/go-sdk/client"
+	"github.com/dapr/kit/logger"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"strconv"
+	"testing"
+	"time"
+)
+
+const (
+	sidecarNamePrefix       = "ravendb-sidecar-"
+	stateStoreName          = "statestore"
+	certificationTestPrefix = "stable-certification-"
+	dockerComposeYAML       = "docker-compose.yml"
+)
+
+func TestRavenDB(t *testing.T) {
+	fmt.Printf("testing started:")
+	log := logger.NewLogger("dapr.components")
+	stateStore := stateRavenDB.NewRavenDB(log).(*stateRavenDB.RavenDB)
+	ports, err := daprTesting.GetFreePorts(2)
+	require.NoError(t, err)
+
+	stateRegistry := stateLoader.NewRegistry()
+	stateRegistry.Logger = log
+	stateRegistry.RegisterComponent(func(l logger.Logger) state.Store {
+		return stateStore
+	}, "ravenDb")
+
+	currentGrpcPort := ports[0]
+	currentHTTPPort := ports[1]
+
+	basicTest := func(ctx flow.Context) error {
+		client, err := daprClient.NewClientWithPort(strconv.Itoa(currentGrpcPort))
+		if err != nil {
+			panic(err)
+		}
+		defer client.Close()
+
+		err = client.SaveState(ctx, stateStoreName, certificationTestPrefix+"key1", []byte("ravenCert1"), nil)
+		require.NoError(t, err)
+
+		err = client.SaveState(ctx, stateStoreName, certificationTestPrefix+"key2", []byte("ravenCert2"), nil)
+		require.NoError(t, err)
```

Comment:
I think we should either get this and verify the value, like we do for the first key, or remove it.